doc/nrf/releases_and_maturity/releases/release-notes-2.8.0.rst

```diff
@@ -0,0 +1,1458 @@
+    The versioning is only applied to the application configurations that use the MCUboot bootloader.
+  * The :ref:`CONFIG_DESKTOP_USB_HID_REPORT_SENT_ON_SOF <config_desktop_app_options>` Kconfig option to :ref:`nrf_desktop_usb_state`.
+    The option allows to synchronize providing HID data with USB Start of Frame (SOF).
+    The feature reduces the negative impact of jitter related to USB polls, but it also increases HID data latency.
+    For details, see :ref:`nrf_desktop_usb_state_sof_synchronization`.
+  * Local HID report buffering in :ref:`nrf_desktop_usb_state`.
+    This ensures that the memory buffer passed to the USB next stack is valid until a HID report is sent and allows to enqueue up to two HID input reports for a USB HID instance (used only when :ref:`CONFIG_DESKTOP_USB_HID_REPORT_SENT_ON_SOF <config_desktop_app_options>` Kconfig option is enabled).
+  * Bootup logs with the manifest semantic version information to :ref:`nrf_desktop_dfu_mcumgr` when the module is used for SUIT DFU and the SDFW supports semantic versioning (requires v0.6.2 and higher).
+  * Manifest semantic version information to the firmware information response in :ref:`nrf_desktop_dfu` when the module is used for SUIT DFU and the SDFW supports semantic versioning (requires v0.6.2 and higher).
+  * A missing DTS node compatible with ``zephyr,hid-device`` to the nRF52840 DK in the MCUboot QSPI configuration.
+    This ensures support for HID over USB when the USB next stack is selected.
+  * The USB next stack (:ref:`CONFIG_DESKTOP_USB_STACK_NEXT <config_desktop_app_options>`) implies partial erase feature of the nRF SoC flash driver (:kconfig:option:`CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE`).
+    This is done to improve stability of the USB next stack.
+    The partial erase feature works around device errors that might be reported by Windows USB host in Device Manager if USB cable is connected while erasing secondary image slot in the background.
+  * Bluetooth connectivity support (:ref:`CONFIG_DESKTOP_BT <config_desktop_app_options>`) implies using a separate workqueue for connection TX notify processing (:kconfig:option:`CONFIG_BT_CONN_TX_NOTIFY_WQ`) if MPSL is used for synchronization between the flash memory driver and radio (:kconfig:option:`CONFIG_SOC_FLASH_NRF_RADIO_SYNC_MPSL`).
+    This is done to work around the timeout in MPSL flash synchronization (NCSDK-29354 known issue).
+    See :ref:`known_issues` for details.
+  * The DVFS module to the nRF Desktop application.
+    You can use the module on the nRF54H20 DK with the ``nrf54h20dk/nrf54h20/cpuapp`` board target.
+    The module is responsible for switching the frequency and voltage according to the application's needs.
+    The module ensures that lowest DVFS frequency and voltage is used unless a compute-intensive operation requests higher frequency and voltage.
+    It listens to Event Manager events and applies DVFS frequency according to the application state.
```

Comment:
@zycz, I guess we need this note only once? (it seems duplicated a few lines below)

Reply:
That's correct we should remove the first one.